packages/iterator-utilities/package.json

```diff
@@ -687,7 +687,7 @@
 		}
 	},
 	"sideEffects": [
-		"./dist/esm/chunk-G5GHKT7C.mjs",
```

Comment:
Undo this change so it doesnt show up in the conventional commit scan as a feat for iterator-utilities

Reply:
Done